infer_structcol/model.py

```diff
@@ -12,6 +12,10 @@
 # define limits of validity for the MC scattering model
 min_phi = 0.35
 max_phi = 0.73
+min_radius = 70. # in nm
```

Comment:
In addition, I believe the Mie solutions are exact, so they could be used for a wider range of particles sizes. Even though Mie is generally only used when particles are comparable to the wavelength of light, it is still valid outside of that. There are just other approximations that are commonly used in those cases. Maybe we should consider extending the range of radii